spec/models/payment_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Payment, type: :model do
```

Comment:
- [ ] When i am running `rspec spec` the following `tests` :point_down:  are failing please fix them

![image](https://user-images.githubusercontent.com/75126481/164932624-af32bf55-69f2-4817-98f4-aca29a5ce7c2.png)